index-en.md

```diff
@@ -475,7 +478,101 @@ css:
 		</li>
 	{% endfor %}
 	</ul>
-
+<h2 id="wetboew" class="mrgn-bttm-lg">Wet-boew</h2>
+	<ul class="row list-unstyled wb-eqht-grd wb-filter mrgn-tp-md pb-4" data-wb-filter='{ "selector": ">li" }'>
+	{% for wetboew in site.data.[ "wet-boew" ] %}
```

Comment:
When I build locally, I got this line throwing the following error:
>     Liquid Warning: Liquid syntax error (line 471): Expected id but found open_square in "wetboew in site.data.[ "wet-boew" ]" in index-en.md

And when I looked at the Github page build log, I found it too.
https://github.com/ouafaaetta/GCWeb/actions/runs/6267244900/job/17019886017#step:4:1248
>     Liquid Warning: Liquid syntax error (line 471): Expected id but found open_square in "wetboew in site.data.[ "wet-boew" ]" in index-en.md